app/screens/integration_selector/integration_selector.tsx

```diff
@@ -190,45 +202,15 @@ function IntegrationSelector(
 
         switch (dataSource) {
             case ViewConstants.DATA_SOURCE_USERS: {
-                setMultiselectSelected((current) => {
-                    const multiselectSelectedItems = {...current};
-
-                    if (current[itemKey]) {
-                        delete multiselectSelectedItems[itemKey];
-                    } else {
-                        multiselectSelectedItems[itemKey] = item as UserProfile;
-                    }
-
-                    return multiselectSelectedItems;
-                });
+                setMultiselectSelected((current) => toggleFromMap(current, itemKey, item as UserProfile));
```

Comment:
You can use the function form also on `handleRemoveOption`.

Reply:
done!